src/components/BookForm/BookForm.js

```diff
@@ -1,13 +1,58 @@
-import React from 'react';
-
-const BookForm = () => (
-  <div>
-    <form>
-      <input type="text" name="title" placeholder="Book title" />
-      <input type="text" name="author" placeholder="Author" />
-    </form>
-    <button type="button">Add Book</button>
-  </div>
-);
+import React, { useState } from 'react';
+import { generate } from 'randomized-string';
+import { useDispatch } from 'react-redux';
+import { addNewBook, addBook } from '../../Redux/Books/Books';
 
+const BookForm = () => {
+  const [bookInit, setBook] = useState({
+    title: '',
+    category: '',
```

Comment:
- [ ] I kindly suggest that you please add author to your state here because it is a part of the requirements to have the `author` in your local  state